src/modules/enrollment/components/EntryExitDatesWithAssessmentLinks.tsx

```diff
@@ -34,6 +34,8 @@ const EntryExitDatesWithAssessmentLinks: React.FC<Props> = ({ enrollment }) => {
 
   const canLinkToIntake =
     enrollment.access.canEditEnrollments || !enrollment.inProgress;
+  const canLinkToExit =
+    enrollment.access.canEditEnrollments || enrollment.exitDate;
```

Comment:
We probably shouldn't link to starting an exit for an enrollment that is WIP, since it can't be exited yet.
```suggestion
    !enrollment.inProgress && (enrollment.access.canEditEnrollments || enrollment.exitDate);
```